scripts/confirm_lb_active.sh

```diff
@@ -3,47 +3,43 @@
 set -euo pipefail
 
 REGION="$1"
-RESOURCE_GROUP_ID="$2"
-LB_ID="$3"
-
-# Expects the environment variable $IBMCLOUD_API_KEY to be set
-if [[ -z "${IBMCLOUD_API_KEY}" ]]; then
-    echo "API key must be set with IBMCLOUD_API_KEY environment variable" >&2
-    exit 1
-fi
+LB_ID="$2"
+PRIVATE_ENV="$3"
+API_VERSION="2024-03-01"
```

Comment:
is the version fixed because related to a specific requirement?
otherwise we could set to (today - 1) to avoid to stick to a fixed value that we have to maintain in the future